src/styles/core/_vars.scss

```diff
@@ -14,7 +14,7 @@ $globalnav-height: 44px;
 $form-focus-size: 4px;
 
 // Border radius
-$border-radius: 4px !default;
+$border-radius: var(--border-radius, 4px) !default;
```

Comment:
This is cool and probably safe for the radius, but we need to careful in the future if we plan to do this to the other vars, as some are used in some SASS math 🤣

Reply:
For sure. Please let me know if you find any potential issues with this—I was thinking the same thing when making this change.

Reply:
Quick search shows these places:

1. https://github.com/apple/swift-docc-render/pull/402/files#diff-0d3d5e64c379c67e30afd3b923825a5f077aa0a1d0bf91b9a3e5c53cd6654a07R46
2. https://github.com/mportiz08/swift-docc-render/blob/improve-theme-settings-support/src/components/Filter/FilterInput.vue#L429
3. https://github.com/mportiz08/swift-docc-render/blob/improve-theme-settings-support/src/components/Filter/FilterInput.vue#L487-L488

Reply:
Thanks for spotting those! The first link isn't working for me, but I'm guessing it was an instance in the badge component where there was a Sass calculation derived from the `$border-radius` var.

I think I've fixed these with minimal changes to use `calc`, which should help the CSS and Sass vars play more nicely together: 2ca4be847d1c06dcdabb4cb9bf00e1139ba46428

In the future, I would love to entirely move away from Sass vars in favor of CSS vars due to issues like this, but that would require a much larger refactor. I also initially tried the reverse of having the CSS var get defined by the SASS var, but that seemed like it would require lots of changes to a huge number of files.

Let me know if you notice any other issues related to this please, thanks!